libraries/utilities/install_keys.py

```diff
@@ -1,14 +1,16 @@
 import os
 import sys
+import json
 
 from optparse import OptionParser
 from generate_clusters_from_pool import get_hosts
 import paramiko
 
 
-def install_keys(public_key_path, user_name, ssh_password):
+def install_keys(public_key_path, user_name, ssh_password, hosts):
```

Comment:
Can you have hosts with default value as 'None'
install_keys(public_key_path, user_name, ssh_password, hosts=None): 
Just in case if this method used any where in the future, it won't force to use hosts field

Reply:
fixed, thanks for reminding